lib/internal/bootstrap/node.js

```diff
@@ -40,6 +40,7 @@ const { internalBinding, NativeModule } = loaderExports;
 const { Object, Symbol } = primordials;
 const config = internalBinding('config');
 const { deprecate } = NativeModule.require('internal/util');
+const { getOptionValue } = NativeModule.require('internal/options');
```

Comment:
Please do not depend on any CLI options during the execution of `lib/internal/bootstrap/node.js` - we are trying to make this file free from any environment states. The descriptors can be overridden later in `pre_execution.js`.

Reply:
Sure, I've moved this logic there.